ppr-ui/src/components/mhrTransfers/BusinessSearchAutocomplete.vue

```diff
@@ -38,27 +43,25 @@
             </div>
           </v-list-item-group>
         </v-list>
-        <v-list v-else-if="!isSearchResultSelected && showDropdown && !searching && autoCompleteResults.length === 0">
-          <v-list-item class="auto-complete-item ">
-            <v-list-item-content class="px-2 pt-1 pb-0">
-              <v-list-item-subtitle>
-                <v-row class="auto-complete-row">
-                  <v-col id="no-party-matches">
-                    <p>
-                      <strong>
-                        No matches found.
-                      </strong>
-                    </p>
-                    <p>
-                      Ensure you have entered the correct, full legal name of the organization before entering the phone
-                      number and mailing address.
-                    </p>
-                  </v-col>
-                </v-row>
-              </v-list-item-subtitle>
-            </v-list-item-content>
-          </v-list-item>
-        </v-list>
+        <div
+          v-else-if="
+            !isSearchResultSelected &&
+              autoCompleteIsActive &&
+              !searching &&
+              autoCompleteResults &&
+              autoCompleteResults.length === 0
+          "
```

Comment:
Not required but a computed value might make this a cleaner template :)

Reply:
Good idea. Will update.